x/ibc/core/03-connection/types/keys.go

```diff
@@ -20,10 +26,34 @@ const (
 	KeyNextConnectionSequence = "nextConnectionSequence"
 
 	// ConnectionPrefix is the prefix used when creating a connection identifier
-	ConnectionPrefix = "connection"
+	ConnectionPrefix = "connection-"
 )
 
 // FormatConnectionIdentifier returns the connection identifier with the sequence appended.
 func FormatConnectionIdentifier(sequence uint64) string {
-	return fmt.Sprintf("%s-%d", ConnectionPrefix, sequence)
+	return fmt.Sprintf("%s%d", ConnectionPrefix, sequence)
```

Comment:
Wait shouldn't this be `%s-%d`?

Reply:
no I updated the ConnectionPrefix to be `connection-`

Reply:
the reason being is I need it for `Split`. I could alternatively add the `-`, but it seemed safe to just include the dash in the prefix

Reply:
ahh right